protobuf/video/sfu/event/events.proto

```diff
@@ -68,10 +68,20 @@ message SfuEvent {
     GoAway go_away = 20;
     // ICERestart tells the client to perform ICE restart.
     ICERestart ice_restart = 21;
+    // PinnedSessionsChanged is sent when the pinned sessions change.
+    PinnedSessionsChanged pinned_sessions_changed = 22;
   }
 }
 
-message Error{
+// PinnedSessionsChanged is sent when the pinned sessions change.
+message PinnedSessionsChanged {
+  // pinned_sessions contains the list of sessions that are pinned
+  // in the call. The order of the sessions in the list is the order
+  // in which they are pinned (descending order of priority).
+  repeated string pinned_sessions = 1;
```

Comment:
Should we call it pinned_user_ids?
remote rpc calls are never session aware so this term is confusing

Reply:
this is just forwarding user input, the client tells the coordinator which session_id to pin and the coordinator fwd the request to the SFU

Reply:
this is not an rpc between coordinator and SFU, this is the event from SFU to clients